_posts/2020-03-15-Sharezone-In-Review-March-Part-1.md

```diff
@@ -0,0 +1,88 @@
+---
+layout: post
+title: "Öffentlichkeitsarbeit, Demo-Accounts und Zukunftsideen [SIR]"
+---
+Durch die klasse Öffentlichkeitsarbeit von Nils, haben wir interessierte Schulen bekommen, denen wir manuell Demoaccounts zur Verfügung gestellt haben.  
+Dass das Anlegen der Accounts noch manuell läuft zeigt eine größere Herausforderung von uns als Gründer auf: Priorisierung.   
+Nebenbei haben wir an einer Zukunftsidee von Sharezone als Tool im Unterricht gebaut.
+
+*In "[Sharezone In Review]({{ site.baseurl }}/Introducing-Sharezone-In-Review)" stellen wir euch vor, was bei uns gerade bei Sharezone abläuft.\
+In dem Beitrag geht es um die ersten beiden Wochen im März.*
+
+## Öffentlichkeitsarbeit
+![]({{ site.baseurl }}/images/Sharezone-In-Review-March+-Part-1/sz-logo.svg)
+
+Bei dem Berg an Aufgaben fällt es uns oft schwer viel Öffentlichkeitsarbeit für Sharezone zu machen. Um so besser, dass Nils in den letzten zwei Wochen aber so richtig reingehauen hat: Von einem Vortrag auf einem Bundeskongress, einem Radio-Beitrag mit zugehörigem Artikel, zu einem Podcast, hat er viel von sich in den letzten zwei Wochen hören und blicken lassen.  
+
+### Vortrag "Warum jede:r ein Startup gründen sollte"
+Nils konnte vor einer versammelten Gruppe von Lehrkräften den Vortrag "Warum jede:r ein Startup gründen sollte". Als Beispiel ging es natülich um das beste Startup der Welt: Sharezone 😉
+
+Doch warum sollte denn jetzt überhaupt jeder ein Startup gründen? Kurzgefasst: Man lernt Gebietsübergreifend (Marketing, Informatik, Vertrieb, Design, Jura, etc.) sehr viele Sachen auf einer Art und Weise, die traditioneller Unterricht nicht vermittelt.    
+
+Der Vortrag wurde im Rahmen der 112. MNU-Bundeskonferenz zur Förderung des MINT-Unterrichts gehalten. Der Vortrag ist zwar (nach unserem Wissen) nicht auf Youtube, eine ältere Version des Vortrags von Nils kann sich aber [hier](https://www.youtube.com/watch?v=Bg0okdOOnZw) angeguckt werden.
+
+### Radio-Beitrag mit Artikel über Pilotschule in Berlin
+![Überschrift und Sub-Überschrift des rbb24-Artikels]({{ site.baseurl }}/images/Sharezone-In-Review-March+-Part-1/rbb-artice-intro.png)
+
+Neben dem Vortrag gabs noch mehr von Nils, diesmal im Doppelpakt. rbb24 veröffentlichte ein Radiosegment und [einen kurzen Artikel](https://www.rbb24.de/panorama/thema/corona/beitraege/2021/03/lern-app-berlin-pilotprojekt-grundschule.html) über eine Berliner Grundschule, welche Teil unseres Pilotschul-Programms ist.  
+
+Mit dem Pilotschul-Programm haben Schulen die Möglichkeit mit uns zusammen Sharezone für die ganze Schule einzurichten und konstenlos zu betreiben.   
```

Comment:
Das klingt bisher noch so, dass es ein offenes Programm ist, an welchem Schulen weiterhin teilnehmen / bewerben können. Vielleicht kann kann sowas hinzufügen: "Letztes Jahr hatten Schulen die Möglichkeit sich auf unser Pilotschul-Programm zu bewerben."

Reply:
Habe haben jetzt in hatten umgeändert, sollte hoffentlich reichen, oder?

Reply:
Ja, passt 👍